include/myst/pubkey.h

```diff
@@ -9,15 +9,6 @@
 #define MYST_PUBKEY_NAME_SIZE 256
 #define MYST_PUBKEY_MAGIC 0x6f14783e80394815
```

Comment:
The above two definitions are not used anymore, right?

Reply:
Good catch! Fixed and pushed.